less/admin/ExtensionWidget.less

```diff
@@ -0,0 +1,86 @@
+.ExtensionsWidget {
+  background-color: @body-bg;
+  padding: 0;
+}
+
+.ExtensionsWidget-list {
+  > .container {
+    padding: 0;
+    background-color: @body-bg;
+
+    .ExtensionList-Category {
+      background: @control-bg;
+      padding: 20px 0 30px 20px;
+      margin-bottom: 20px;
+      border-radius: @border-radius;
+
+      .ExtensionList-Label {
+        margin-top: 0;
+        color: @muted-color;
+      }
+    }
+
+    .ExtensionGroup {
+      margin-bottom: 20px;
+
+      h3 {
+        color: @muted-color;
+        text-transform: uppercase;
+        font-size: 12px;
+        margin: 0 0 10px;
+      }
+    }
+
+    .ExtensionList {
+      margin: 0 -6px;
```

Comment:
Since we decided to go for grid, we can drop these margins that I had suggested with flex